src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -99,6 +108,16 @@ end
 
 local function tabulateCommon( bPaged, ... )
     local tAll = { ... }
+    for i=1,#tAll,2 do
```

Comment:
This is still wrong... the API doesn't require you to go "number, table, number, table". you can use any combination.